packages/api-server/src/methods/types.ts

```diff
@@ -16,7 +16,16 @@ export type Response = number | string | boolean | SyningStatus | Array<string>;
 
 export type Callback = (err: Error, res?: any | Response) => void;
 
-export type BlockParameter = HexNumber | "latest" | "earliest" | "pending";
+export type BlockTag = "latest" | "earliest" | "pending";
+
+// Eip1898 support block hash and block number
+export interface BlockSpecifier {
```

Comment:
What about renaming EIP1898BlockSpecifier ?

Reply:
a bit too long. I personally think `BlockSpecifier` with commenting eip1898 is clear enough?